scripts/day2/edge-save-images.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+list="edge-release-images.txt"
+images="edge-images.tar.gz"
+source_registry=""
+
+usage () {
+    echo "USAGE: $0 [--image-list edge-release-images.txt] [--images edge-images.tar.gz] --source-registry registry.suse.com"
+    echo "  [-i|--images] tar.gz file to use with 'docker load'."
+    echo "  [-s|--source-registry] source registry to pull images from in registry:port format e.g. registry.suse.com."
+    echo "  [-l|--image-list path] text file with list of images; one image per line."
+    echo "  [-h|--help] Usage message"
+}
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -i|--images)
+        images="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -s|--source-registry)
+        if [ -z "${source_registry}" ]; then
+            source_registry=$2
+        else
+            source_registry="${source_registry},$2"
+        fi
+        shift # past argument
+        shift # past value
+        ;;
+        -l|--image-list)
+        list="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -h|--help)
+        help="true"
+        shift
+        ;;
+        *)
+        usage
+        exit 1
+        ;;
+    esac
+done
+
+if [[ $help ]]; then
+    usage
+    exit 0
+fi
+
+if [[ -z "${images}" ]]; then
+    echo "Missing '-i|--images'"
+    usage
+    exit 1
+fi
+
+if [ -z "${source_registry}" ]; then
+    echo "Must have at least one '-s|--source-registry' entry"
+    usage
+    exit 1
+fi
+
+pulled=""
+while IFS= read -r i; do
+    [ -z "${i}" ] && continue
+
+    found=""
+    for reg_entry in ${source_registry//,/ } ; do
+        img="${reg_entry}/${i}"
+
+        if docker pull "${img}" > /dev/null 2>&1; then
```

Comment:
same than before (if you accept the podman, please check the rest of docker commands)

Reply:
Yea, I changed it to podman at the docs, but wanted to have a discussion whether we need the scripts first before doing the changes in the scripts. Otherwise I agree.